app/main/views/briefs.py

```diff
@@ -81,6 +83,10 @@ def ask_brief_clarification_question(brief_id):
 def start_brief_response(brief_id):
     brief = get_brief(data_api_client, brief_id, allowed_statuses=['live'])
 
+    if not (datetime.strptime(current_app.config['FEATURE_FLAGS_NEW_SUPPLIER_FLOW'], "%Y-%m-%d")
+            <= datetime.strptime(brief['publishedAt'][0:10], "%Y-%m-%d")):
```

Comment:
I found another way! Have imported the constant from dmutils that is used to convert the `published_at` date of the brief to string when serializing it in the api. It can be used to do the reverse and create a date object in the supplier frontend.